administrator/components/com_finder/src/View/Item/HtmlView.php

```diff
@@ -12,6 +12,7 @@
 
 use Joomla\CMS\Language\Text;
 use Joomla\CMS\MVC\View\HtmlView as BaseHtmlView;
+use Joomla\CMS\Toolbar\Toolbar;
```

Comment:
```suggestion
use Joomla\CMS\Router\Route;
use Joomla\CMS\Toolbar\Toolbar;
```
![image](https://github.com/joomla/joomla-cms/assets/66922325/05385a5e-e442-4645-8d4e-5d1ad6997a62)

Reply:
thanks for the finding